packages/video_player/video_player_web/pubspec.yaml

```diff
@@ -22,7 +22,7 @@ dependencies:
   flutter_web_plugins:
     sdk: flutter
   video_player_platform_interface: ^6.2.0
-  web: ">=0.5.1 <2.0.0"
+  web: ^1.0.0
```

Comment:
If `web/helpers.dart` was already unused, we shouldn't need to make this change.

Reply:
ack